### DISCUSSION TITLE
README does not correctly render Markdown tables

### DISCUSSION THREAD

**Discussion by cachedout** [community/community#20034] [Projects and Issues]
To reproduce:

1. Create a Markdown table in a project board README which contains two columns. 
2. In the first column put a word with 5-10 characters.
3. In the second column, put 5-10 works of around 5 characters each.
4. Note that the second column gets cut off in the README when viewing it from the project.

Screenshot:
<img width="392" alt="Screenshot 2022-07-05 at 15 15 20" src="https://user-images.githubusercontent.com/111616/177336376-4503616d-960c-4ee8-8270-5ef0611e6d11.png">

**Comment by diegoleondev** [comment]
Hello, this is completely normal, the tables in do not adapt well to a small space, Try other ways to render the content if that view is needed

<details>
<summary>🔵Ready</summary>
...
</details>

``` 
<details>
<summary>🔵Ready</summary>
...
</details>
```

**Reply by cachedout** [reply]
I'm sorry, but I disagree. This is a bug. The slide-out should adapt to the content, not the other way around.
